chapters.md

```diff
@@ -17,7 +17,62 @@ As an example, consider a `Parent ChapterAtom Element` that has its `ChapterFlag
 
 Three `Edition Flags` are defined to describe the bevahior of the `EditionEntry Element`: `EditionFlagHidden`, `EditionFlagDefault` and `EditionFlagOrdered`.
```

Comment:
> (in which case the default value of the Element applies, which is `false`)

I would mention this above, after the sentences on line 12, instead of as a parens here (just my opinion).